services/src/main/java/org/fao/geonet/api/records/MetadataWorkflowApi.java

```diff
@@ -704,7 +712,12 @@ public List<MetadataStatusResponse> getWorkflowStatusByType(
             if (!CollectionUtils.isEmpty(recordIdentifier)) {
                 for (Integer recordId : recordIdentifier) {
                     try {
-                        ApiUtils.canEditRecord(String.valueOf(recordId), request);
+                        if (allowedAccessLevelProfile == Profile.RegisteredUser) {
```

Comment:
With this change, when selecting the `RegisteredUser` access level, users will be able to see the history of metadata that they can view, while selecting other values, only the metadata they can edit.

I think at least should be added some explanation in https://github.com/geonetwork/core-geonetwork/pull/7450/files#diff-e312a431007c32b1300d73d63418ba4ef8154f864ce4734d7461822738478ca9R886

Also at the end of this manual page a section describing the feature would be good: https://docs.geonetwork-opensource.org/4.2/administrator-guide/configuring-the-catalog/system-configuration/

Reply:
I have update both help text, also add the the section in system-configuration.md